agent/go.mod

```diff
@@ -12,6 +12,7 @@ require (
 	github.com/go-playground/validator/v10 v10.5.0
 	github.com/google/martian v2.1.0+incompatible
 	github.com/gorilla/websocket v1.4.2
+	github.com/op/go-logging v0.0.0-20160315200505-970db520ece7 // indirect
```

Comment:
indirect

Reply:
@RoyUP9 https://github.com/up9inc/mizu/commit/e554e40f4a16f07bf7b1f2ee4a7d2544a5a5a19d removes this line.